intermediate-algorithm-scripting-yahtzee-game/index.html

```diff
@@ -1,104 +1,126 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Simplified Yahtzee Dice Game</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
 
-<head>
-  <meta charset="utf-8" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Simplified Yahtzee Dice Game</title>
-  <link rel="stylesheet" href="css/style.css">
-  <script defer src="js/index.js"></script>
-</head>
+  <body>
+    <header>
+      <h1>Yahtzee!</h1>
+    </header>
 
-<body>
-  <header>
-    <h2>Welcome to a simplified version of Yahtzee. Amazon's #1 best-selling dice game.</h2>
-
-    <p>
-      <a href="https://www.wikihow.com/Play-Yahtzee" target="_blank">
-        If you're not familiar with the game, check out the rules here.
-      </a>
-    </p>
-
-    <p>
-      The rules are the same as in the above-linked article with some changes as follows to make the game a bit
-      simpler:
-    </p>
-
-    <ul>
-      <li>There will be 6 rounds or turns, each comprised of three dice rolls</li>
-      <li>Score combinations will be only from the lower section</li>
-      <li>No bonus scores of any kind</li>
-    </ul>
+    <main>
+      <div id="game">
+        <div id="dice">
+          <button class="die selected"></button>
+          <button class="die selected"></button>
+          <button class="die selected"></button>
+          <button class="die selected"></button>
+          <button class="die selected"></button>
+        </div>
 
-    <p>The green boxes are selected dice.
-      You can
-      select / unselect a die by clicking on it. <br> On the first roll of every round you need to roll all the five
-      dice. Now, let's get rolling!</p>
-  </header>
+        <p class="rounds-text">
+          <strong>Rolls:</strong> <span id="current-round-rolls">0</span> |
+          <strong>Round:</strong> <span id="current-round">1</span>
+        </p>
 
-  <hr>
+        <div id="score-options">
```

Comment:
At first I was thinking that this might also be better as a `fieldset` element to group these radio inputs, but it's probably fine the way it is. The fact that all these elements are part of the same form should be enough to group them together.

Reply:
I personally like keeping it as a div because it provides more flexibility for styling options.